sympy/physics/mechanics/particle.py

```diff
@@ -157,20 +162,22 @@ def angular_momentum(self, point, frame):
 
         """
 
-        return self.point.pos_from(point) ^ (self.mass * self.point.vel(frame))
+        return cross(self.point.pos_from(point),
+                     (self.mass * self.point.vel(frame)))
 
     def kinetic_energy(self, frame):
-        """Kinetic energy of the particle.
+        r"""Kinetic energy of the particle.
 
         Explanation
         ===========
 
-        The kinetic energy, T, of a particle, P, is given by
+        The kinetic energy, $T$, of a particle, ``P``, is given by
```

Comment:
Removing commas for consistency with other docstrings.
```suggestion
        The kinetic energy $T$, of a particle ``P``, is given by
```